cristin-import/src/test/java/cucumber/CristinContributorTransformer.java

```diff
@@ -0,0 +1,38 @@
+package cucumber;
+
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicInteger;
+import no.unit.nva.cristin.CristinDataGenerator;
+import no.unit.nva.cristin.mapper.CristinContributor;
+import no.unit.nva.cristin.mapper.CristinContributor.CristinContributorBuilder;
+
+public class CristinContributorTransformer {
```

Comment:
Helper calss for Creating CristinContributors from Cucumber Datatable instances. Both this and the previous class do not follow exactly Cucumber's `@DataTableType` entry because we want to have multiple transformation methods for the different scenarios.
`